_layouts/seminar.html

```diff
@@ -49,11 +49,20 @@ <h1 class="page-title">
     <h2>Papers and Slides</h2>
 
 {% assign slidedistinctions = page.papers | map: "distinctionslides" | compact %}
-
 {% assign paperdistinctions = page.papers | map: "distinctionpaper" | compact %}
 
 {% if slidedistinctions contains true or paperdistinctions contains true %}
-<p>Papers and presentations that were evaluated as deserving a distinction are marked with &#x2B50.</p>
+  {% assign star_items = "" | split: "," %}
```

Comment:
Is this just a neat way to create an empty array?

Reply:
To my knowledge, it's the only way to create an empty array in Jekyll.